task-1/api/user.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    IRouter,
+} from "express";
+import {
+    RequestHandler,
+} from "express-serve-static-core";
+import * as uuid from "uuid";
+
+import {
+    User,
+} from "../types";
+
+
+const validateUser: RequestHandler = (req, res, next) => {
+    if (res.locals.user === undefined) {
+        res.status(404).json({ message: `User with id ${req.params.id} not found.` });
+        return;
+    }
+
+    next();
+}
+
+export default (router: IRouter, usersStorage: Array<User>) => {
+    router.param("id", (req, res, next, id) => {
+        res.locals.user = usersStorage.find((user) => user.id === id);
+        next();
+    });
+
+    router.get("/:id", validateUser, (req, res) => {
+        res.json({ body: res.locals.user, message: "ok" });
```

Comment:
let's filter out `isDeleted` field from all endpoinds that return it now

Reply:
Done